pkg/cmd/pr/status/status_test.go

```diff
@@ -127,6 +138,31 @@ func TestPRStatus_reviewsAndChecks(t *testing.T) {
 	}
 }
 
+func TestPRStatus_reviewsAndChecksWithStatesByCount(t *testing.T) {
```

Comment:
I chose to move forward with this PR because it's been open a while and then to revisit with moving these tests to the newer style. This test is a copy of the one above but returning a response with the new fields.